canonical-web-and-design/snapcraft.io
# Feature branch - distro page

Fixes https://github.com/CanonicalLtd/snap-squad/issues/1017

Adds a distro install page for snaps.
This is a feature branch, all the features and code here was reviewed in smaller PRs before.

Purpose of this PR is to give it final QA and release to master.

### QA
- Visit distro page for all the distros and make sure they render fine (try with different snaps):

https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/riot-web/arch
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/pick-colour-picker/centos
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/wing7/debian
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/eric-ide/elementary-os
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/rider/fedora
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/emacs/kde-neon
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/toontown/kubuntu
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/scrn/manjaro
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/eric-ide/mint
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/natron/opensuse
https://snapcraft-io-canonical-web-and-design-pr-1955.run.demo.haus/install/marsshooter/ubuntu


<img width="728" alt="Screenshot 2019-06-10 at 14 36 51" src="https://user-images.githubusercontent.com/83575/59195816-38bbeb80-8b8d-11e9-835d-26898dfd708a.png">
